src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+	<PropertyGroup>
+		<PackageId>AlertsManagement.Tests</PackageId>
+		<Description>AlertsManagement.Tests Class Library</Description>
+		<AssemblyName>AlertsManagement.Tests</AssemblyName>
+		<VersionPrefix>1.0.0</VersionPrefix>    
+	</PropertyGroup>
+
+  <ItemGroup>
+	<PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <ProjectReference Include="..\Management.AlertsManagement\Microsoft.Azure.AlertsManagement.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+
+  <PropertyGroup>
```

Comment:
@mahakjain314 you do not need this, it's being added implicitly to all test projects.
This way we can control the target Fx version from a central location.

Reply:
Makes sense. Removed the same.